vm/systemSmartContracts/auction.go

```diff
@@ -92,8 +88,7 @@ func NewStakingAuctionSmartContract(
 		return nil, vm.ErrNilEpochNotifier
 	}
 
-	baseConfig := AuctionConfig{
-		NumNodes:    uint32(args.NumOfNodesToSelect),
```

Comment:
The removal of the NumNodes config value will break the backwards compatibility?

Reply:
Yes it is. This config was not saved to storage.